packages/compass-indexes/src/components/search-index-template-dropdown/index.tsx

```diff
@@ -14,18 +14,18 @@ import {
 const containerStyles = css({
   display: 'flex',
   flexDirection: 'column',
-  gap: spacing[1],
+  gap: spacing[100],
 });
 
 const dropdownLabelStyles = css({
   display: 'flex',
-  gap: spacing[1],
+  gap: spacing[100],
   alignItems: 'center',
 });
 
 type SearchIndexTemplateDropdownProps = {
   tooltip: string;
-  isVectorSearchSupported?: boolean;
+  isVectorSearchSupported: boolean;
```

Comment:
`?: boolean` is actually a pretty common pattern in our code where we make opt-in things optional for convenience. Unless that's really a required field. So it would be hard to come up with a linting rule that works in all cases.